### ISSUE TITLE
send notification to opsgiene

### ISSUE THREAD

**Issue by amolsh** [draios/python-sdc-client#17]
Hi,
I wanted to send notification to OpsGenie , i declared like this and it's not working
{'type': 'OPSGENEI', 'channel': 'Cloud Platform OpsGenie'}

**Comment by philrz** [comment]
@amolsh,

I just merged a fix such that OpsGenie Notification Channels should now be supported. At this point, using the same `examples/create_alert.py` that you were starting from, if I find the channel with:

```
notify_channels = [ {'type': 'OPSGENIE', 'name': 'Cloud Platform OpsGenie'} ]
```

...the same code fragment in your original issue now works for me, with the resulting Alert as seen in my UI:

![image](https://cloud.githubusercontent.com/assets/5934157/23812666/548de0f0-0590-11e7-9eaf-6547353688ab.png)

In conclusion, it looks to me like your second reported issue was the whole of the issue. Can you confirm if that's the case? If you could, please just pull the latest code from GitHub. I'll upload a release with the OpsGenie fix to PyPI for `pip install` use once I'm confident I've addressed all your issues.

**Comment by philrz** [comment]
@amolsh,

As it turns out, I had to tag and upload a new version of the Python client onto PyPI for other reasons, so the OpsGenie support should be in there if you just do a `pip install sdcclient` (which should give you version `0.5.1`). Please let me know if your issue is addressed.

**Comment by amolsh** [comment]
@philrz ,
Thanks for help!
Sorry, I couldn't reply early. But, yeah the opsgeine feature is working well.  
You are right, About filters, that's whole new issue. So, should I update the issue for only opsgiene and create separate issue for filters?

**Comment by amolsh** [comment]
@philrz,
I have updated this issues for only opsgiene. And created new issue for filters. issue: #21
